apps/blockchain/test/blockchain/state_test.exs

```diff
@@ -11,6 +11,9 @@ defmodule Blockchain.StateTest do
     stCallCodes: [
       :callcall_00,
       :callcode_checkPC
+    ],
+    stRefundTest: [
+      :refund_singleSuicide
```

Comment:
so currently not all tests are passing?

Reply:
Nope, not yet. I [added](https://github.com/poanetwork/mana/blob/feature/refunds/apps/blockchain/test/blockchain/state_test.exs#L22-L38) and commented the failing tests and will add issues for passing the remaining tests.